src/app/Components/sign-up/sign-up.component.ts

```diff
@@ -11,14 +11,39 @@ import { IUser, CognitoService } from '../../Services/cognito.service';
 export class SignUpComponent {
   loading: boolean;
   user: IUser;
+  public errorInSignUp: string = '';
+  public errorMessage: string = '';
 
   constructor(private router: Router, private cognitoService: CognitoService) {
     this.loading = false;
     this.user = {} as IUser;
   }
 
-  public signUp(): void {
-    this.loading = true;
+  passwordValidate(password: string): void {
+    const regexPassword = /^[\S]+.*[\S]+$/;
+    let validatePassword = (<HTMLInputElement>(
+      document.getElementById('confirmPassword')
+    )).value;
+
+    if (!regexPassword.test(password) && password.length > 5) {
```

Comment:
is this supposed to be `if(!(regexPassword.test(password) && password.length > 5))`: missing parenthesis?